src/lib/journey/callbacks/username/validated-create-username.svelte

```diff
@@ -33,13 +35,17 @@
      *********************************************************************** */
     callback.setInputValue((event.target as HTMLInputElement).value);
   }
+
+  $: {
+    console.log(callback);
+  }
 </script>
 
 <Input
   errorMessage={validationFailure}
```

Comment:
I saw some weird behaviour on this field - the message "Please check this value" was repeated four times, presumably because there are four (different) validation failures, e.g:

"{ \"policyRequirement\": \"REQUIRED\" }",
"{ \"params\": { \"minLength\": 1 }, \"policyRequirement\": \"MIN_LENGTH\" }",
"{ \"params\": { \"minLength\": 1 }, \"policyRequirement\": \"MIN_LENGTH\" }",
"{ \"params\": { \"maxLength\": 255 }, \"policyRequirement\": \"MAX_LENGTH\" }"